src/commands/commands.module.ts

```diff
@@ -1,6 +1,6 @@
 import { Module } from '@nestjs/common';
-import { SynchronizeContributorCommand } from './synchronize-contributor.command';
-import { SynchronizationModule } from '../synchronization/synchronization.module';
+import { SynchronizeContributorCommand } from '@/commands/synchronize-contributor.command';
```

Comment:
```suggestion
import { SynchronizeContributorCommand } from './synchronize-contributor.command';
```

imports from external modules should be absolute
imports from the current module should be relative